packages/nextra-theme/package.json

```diff
@@ -43,13 +43,13 @@
     "react-use": "^17.4.0"
   },
   "devDependencies": {
-    "@edgeandnode/gds": "^2.8.1",
-    "@edgeandnode/organisms": "^2.9.1",
+    "@edgeandnode/gds": "^2.9.0",
+    "@edgeandnode/organisms": "^2.10.0",
     "@emotion/react": "^11.11.1",
     "@types/lodash": "^4.14.196",
-    "@types/react": "^18.2.18",
+    "@types/react": "^18.2.19",
     "@types/react-dom": "^18.2.7",
-    "next": "^13.4.12",
+    "next": "13.4.12",
```

Comment:
There's [an issue with Next 13.4.13](https://github.com/vercel/next.js/issues/53171), so locking the version for now.